provision/vvv-init.sh

```diff
@@ -54,6 +55,31 @@ fi
 cp -f "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf.tmpl" "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf"
 sed -i "s#{{DOMAINS_HERE}}#${DOMAINS}#" "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf"
 
+
+# Proxy images from live site
+if [[ "${LIVE_URL}" != http* ]]; then
+	LIVE_URL=''
+else
+	LIVE_URL=$(cat <<'END_HEREDOC'
+
+
+	# Directives to send expires headers and turn off 404 error logging.
+	location ~* .(js|css|png|jpg|jpeg|gif|ico)$ {
+		expires 24h;
+		log_not_found off;
+		try_files \$uri \$uri\/ @production;
+	}
+	
+	location @production {
+		resolver 8.8.8.8;
+		proxy_pass $LIVE_URL\/\$uri;
+	}
+END_HEREDOC
+)
+fi
+
+sed -i "s,nginx-wp-common.conf;,nginx-wp-common.conf;${LIVE_URL},g" "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf"
```

Comment:
Not sure why `nginx-wp-common.conf` is being referenced here, only `vvv-nginx.conf` should be referenced

Reply:
I used `nginx-wp-common.conf` kind of placeholder here, so that `sed` knows where the `$LIVE_URL` should be inserted. `nginx-wp-common.conf` is almost the last letters in `vvv-nginx.conf`-file, so the images proxy -part will go to the end of file before closing `}`.

But I think the pattern you have since used for example for `{{TLS_CERT}}` (https://github.com/Varying-Vagrant-Vagrants/custom-site-template/blob/master/provision/vvv-init.sh#L59) seems better solution to me 👍

Reply:
Indeed a `{{LIVE_URL}}` would be better, or rather a more generic identifier that gets replaced by a variable from bash